src/wizard/deploy.jl

```diff
@@ -454,7 +454,7 @@ function github_deploy(state::WizardState)
                 print_travis_deploy(f, gr, secure_key)
             end
         catch e
-            Base.display_error(STDERR, e, catch_backtrace())
+            Base.display_error(Compat.stderr, e, catch_backtrace())
```

Comment:
Does this need to be `Compat.stderr`?  Can it just be `stderr`?

Reply:
done